### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mhamidjamil**
- [x] The implementation fulfills the desired requirement.
  <details>
    <summary>Details</summary>
    The code does the intended while also ensuring that it does not cause side-effects or compromise the system performance and functionality.
  </details>
- [x] The code does not add unnecessary complexity.
  <details>
    <summary>Details</summary>
    The code is neither too simple and long (over-engineering) nor too short and complex for peers. With a complex code, devs are likely to introduce bugs when calling or modifying the code.
  </details>
- [x] The code is style guide compliant.
  <details>
    <summary>Details</summary>
    The style guide of project is followed e.g. linting, formatting, types, and other standards. Any style point differing from the guide should not be combined with other changes, as it makes it hard to see what is being changed.
  </details>
- [x] The code follows agreed design principles.
  <details>
    <summary>Details</summary>
    SOLID principles are used efficiently, making the code reusable and scalable.
  </details>
- [x] The code is readable.
  <details>
    <summary>Details</summary>
    The code is readable and comprehensive. Comments are clear, useful, and explain why instead of what.
  </details>
- [x] The code has appropriate tests.
  <details>
    <summary>Details</summary>
    Tests should be added, as appropriate, in the same changelist as the code. Tests should be correct, sensible, and useful.
  </details>
- [x] Documentation has been updated.
  <details>
    <summary>Details</summary>
    Devs should update associated documentation. If code is deleted or deprecated, then the documentation should also be deleted.
  </details>

### CODE REVIEW

File: v5/v5_stable.ino [hunk 1]
```diff
@@ -1,6 +1,6 @@
-//$ last work 23/Oct/23 [10:44 PM]
-// # version 5.5.6
-// # Release Note : Module can now use time send by orange pi
+//$ last work 07/Nov/23 [11:35 PM]
+// # version 5.6.2
+// # Release Note : Unknow message is no readable by orangePi
```
Comment (wasey-rao) [inline v5/v5_stable.ino:3]: Correct comment message and data and time when you pushed the code

File: v5/v5_stable.ino [hunk 2]
```diff
@@ -133,6 +137,7 @@ int allowed_debugging[7] = {0, 0, 1, 0, 1, 0, 1};
 // (debuggerID == 4) // debugging WHATSAPP_debug functionality
 // (debuggerID == 5) // debugging BLE_debug functionality
 // (debuggerID == 6) // debugging SPIFFS_debug functionality
+// (debuggerID == 7) // debugging Orange-pi functionality
```
Comment (wasey-rao) [inline v5/v5_stable.ino:140]: I new task is assigned kindly mark that too.

**Review by wasey-rao — CHANGES_REQUESTED** [review]

File: v5/v5_stable.ino (hunk 1 above)
Comment (mhamidjamil) [reply v5/v5_stable.ino]: resolved

File: v5/v5_stable.ino (hunk 2 above)
Comment (mhamidjamil) [reply v5/v5_stable.ino:140]: kindly mention it, or write details here, as this part is doing its work

**Comment by mhamidjamil** [comment]
also solved #103 #130 #131 #132

**Pull request merged**